src/styles/blocks/card.css

```
@@ -0,0 +1,60 @@
+.card {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 166px;
+  height: 356px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__computerImage {
+  width: 160px;
+  height: 134px;
+  padding: 0 3px 40px;
```

remove padding for pictures, use flex to center the image
```suggestion
  align-self: center;
```

---

add margin-bottom to image to make distance from card title